app/models/admin_set.rb

```diff
@@ -1,5 +1,7 @@
 # There is an interplay between an AdminSet and a PermissionTemplate.
 # @see Hyrax::AdminSetBehavior for further discussion
 class AdminSet < ActiveFedora::Base
+  DEFAULT_ID = 'admin_set/default'.freeze
```

Comment:
I wonder if we might want to move this constant into the `AdminSetBehavior` module so that if folks override the `AdminSet` class downstream (presumably continuing to include the behavior module), the `AdminSet::DEFAULT_ID` constant should still work.